dist/threads/threads.xs

```diff
@@ -421,7 +421,7 @@ STATIC const MGVTBL ithread_vtbl = {
     ithread_mg_free,    /* free */
     0,                  /* copy */
     ithread_mg_dup,     /* dup */
-#if (PERL_VERSION > 8) || (PERL_VERSION == 8 && PERL_SUBVERSION > 8)
+#if PERL_VERSION_GT(5,8,8)
```

Comment:
The better check here would be
```
#ifdef MGf_LOCAL
```

Reply:
Or just leave it out entirely, on partial initialization C will initialize the rest to 0 anyway

Reply:
I'm sorry I don't follow here. Are you suggesting dropping compatibility with < 5.8.7? IMO the optimization you're suggesting should go into blead and not be a part of these changes.

Reply:
I'm just preserving the existing logic here. If you think this should be changed to something else, please open a case for it. I think this outside of the current scope: bumping the version to `7.1.0`

Reply:
> I'm sorry I don't follow here. Are you suggesting dropping compatibility with < 5.8.7? IMO the optimization you're suggesting should go into blead and not be a part of these changes.

I'm suggesting to not explicitly initialize it, because the default initialization in this case will be `0` anyway.